public/javascript/ability-scores.js

```diff
@@ -98,13 +98,15 @@ async function createAbilityScores () {
                 'Content-Type': 'application/json'
             }
         })
+        
         if (response.ok) {
+            const success = await response.json();
```

Comment:
Forcing a wait for response, so the API gateway doesn't reject requests thinking it's spam and  to ensure we get the info before trying next one.